lib/v2/hackertalk/index.js

```diff
@@ -0,0 +1,25 @@
+const got = require('@/utils/got');
+const md = require('markdown-it')();
+
+module.exports = async (ctx) => {
+    const limit = ctx.params.limit || 25;
```

Comment:
```suggestion
    const limit = ctx.query.limit ?? 25;
```
https://docs.rsshub.app/parameter.html#tiao-shu-xian-zhi